python/graphstorm/config/argument.py

```diff
@@ -2076,6 +2076,14 @@ def eval_metric(self):
 
         return eval_metric
 
+    @property
+    def eval_k(self):
```

Comment:
Is it possible to have a list of ks instead of just 1 k?

Reply:
I think most of the LP/retrieval datasets only select one specific k.